src/python/pants/backend/python/rules/pex.py

```diff
@@ -221,6 +222,7 @@ def __init__(
         self,
         *,
         output_filename: str,
+        distributed_to_users: bool,
```

Comment:
Open to suggestions on a better name.

I intentionally did not set a default value. The convenient default would be `True`, but that's not a safe default and could result in subtle bugs. Meanwhile, a default of `False` would be safe, but would risk us missing a very helpful performance optimization. So, I reasoned that it's best to make the call site make an intentional decision.

Reply:
Mm: another potential keyword here is "portable"? More important than the name though is probably a good solid docstring explaining the effect of the flag.

Reply:
Re name, how about "internal_only"? But you can also leave it as-is, main thing is the documentation. No need to bikeshed too much.

Reply:
Sure, I like `internal_only`. Thanks.